docs/reference/execution-providers/CUDA-ExecutionProvider.md

```diff
@@ -0,0 +1,38 @@
+---
+title: CUDA
+parent: Execution Providers
+grand_parent: Reference
+nav_order: 1
+---
+
+# CUDA Execution Provider
+
+The CUDA Execution Provider enables hardware accelerated computation on CUDA-enabled GPUs.
+
+## Build
+For build instructions, please see the [BUILD page](../../how-to/build.md#CUDA).
+
+## Configuration Options
+The CUDA EP supports the following configuration options:
+
+### device_id
+The device ID.
+
+### cuda_mem_limit
+The memory limit of the device memory arena in bytes.
```

Comment:
This might need more clarification based on this question https://github.com/microsoft/onnxruntime/issues/6215.